### PR TITLE
Basic Login Page

### PR DESCRIPTION
**Pull request by naveensaigit**
I have designed a basic login page which redirects to basic.html if email ends with @students.iitmandi.ac.in and has a valid roll no.

### CODE REVIEW

File: templates/login.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        padding: 10px 20px;
+        border: 1px solid #17a2b8;
+        border-radius: 3px;
+        //background:linear-gradient(180deg, rgb(49, 58, 131),rgb(151, 71, 163));
+      }
+      .button{
+        border-radius:30px;
+        background:rgb(255, 255, 255);
+        padding:10px 40px;
+        color:color:rgb(139, 143, 140);
```
Comment (Milind712000) [inline templates/login.html]: invalid css

**Review by Milind712000 — CHANGES_REQUESTED** [review]
opening the app gives an error as basic.html is not in the templates folder
- [x] Rename basic.html and all its references to index.html and move it to templates folder
- [x] Aslo seperate your css and put it in a different file
- [x] Create a layout.html and extract the commonly repeated html to it (make use of Jinja2)
- [ ] the flash message "You have logged-in successfully" cannot be seen currently
- [ ] You used a dependency called wtfforms add the dependency to requirements.txt

**Comment by Milind712000** [comment]
I'll have requirements.txt made and commited in a different PR
till then focus on fixing the other issues

**Comment by naveensaigit** [comment]
I have renamed basic.html to index.html. 
I have created a layout.html. 
I did separate the css, but using an external stylesheet is not working in my system. Please take a look at this.
Lastly, I am unable to figure out why the flash message is not working. Please take a look at this too

**Comment by naveensaigit** [comment]
I have updated everything.
CSS works fine.
The flash message is now displayed correctly for successful as well as unsuccessful login.

**Pull request closed without merging**